app/assets/stylesheets/forms.scss

```diff
@@ -84,3 +84,38 @@ input {
 h2 {
   text-align: center;
 }
+
+.fileUpload {
```

Comment:
Would `.file-upload` work here? It's more according to the class naming conventions.